service/src/main/resources/policydb/changesets/20220909_remove_predecessor.yaml

```diff
@@ -0,0 +1,8 @@
+databaseChangeLog:
+  - changeSet:
+      id: remove predecessor
+      author: dd
+      changes:
+      - dropColumn:
```

Comment:
You don't need to wait for the code changes in this PR (that stop using the column) to be deployed first?